pkg/secrets/passdriver/passdriver.go

```diff
@@ -126,18 +116,18 @@ func (d *Driver) Lookup(id string) ([]byte, error) {
 		return nil, err
 	}
 	if err := d.gpg(context.TODO(), nil, out, "--decrypt", key); err != nil {
-		return nil, fmt.Errorf("%s: %w", id, errNoSecretData)
+		return nil, fmt.Errorf("%s: %w", id, define.ErrNoSuchSecret)
```

Comment:
It doesn't change the current behavior but I am not sure this should return this error here.